src/Stratis.Bitcoin.Features.WatchOnlyWallet/WatchOnlyWalletManager.cs

```diff
@@ -36,6 +37,13 @@ public class WatchOnlyWalletManager : IWatchOnlyWalletManager
         /// <summary>Provider of date time functionality.</summary>
         private readonly IDateTimeProvider dateTimeProvider;
 
+        /// <summary>
+        /// Provides a rapid lookup of transactions appearing in the watch-only wallet.
+        /// This includes both transactions under watched addresses, as well as stored
+        /// transactions.
+        /// </summary>
+        internal Dictionary<uint256, TransactionData> txLookup;
```

Comment:
This probably can be private instead of internal

Reply:
Done